### DISCUSSION TITLE
commits and bcommits default pickers not work

### DISCUSSION THREAD

**Discussion by xixiaofinland** [ibhagwan/fzf-lua#1221] [Q&A] [answered]
In my MacOS Nvim v0.10, fzf-lua `commits` and `bcommits` pickers show briefly and then disappear. 
The nvim messages recorded the warning: "[Fzf-lua] fzf error 2: bind action not specified:  fnc_id=1 })' "
The git has latest version v2.45.1 and executing these commands manually in console work well:
```
git log --pretty=oneline --abbrev-commit --decorate=short
git log --pretty=oneline --abbrev-commit --decorate=short -- <file>
```
`git_files`, `git_tags` and `git_branches` pickers work well.

Be noted that I use identical dotfiles in ubuntu and MacOS, this issue appears in MacOS only while ubuntu displays the pickers correctly.

In case it helps, here is [my fzf-lua config file](https://github.com/xixiaofinland/dotfiles/blob/main/.config/nvim/lua/plugins/fzf-lua.lua)

**Comment by ibhagwan** [comment]
For this it would’ve been better you filed a bug report, there are leading questions in the issue template that help me identify the issue.

- What is your fzf version?
- Can you run `:FzfLua git_commits debug=true` and then post the command that appeared in `:messages`?

**Comment by xixiaofinland** [comment, marked as answer]
@ibhagwan  it was indeed fzf version... 
I had v0.30, `brew install fzf` bumped the version to v0.52.1, now the pickers are working!

**Reply by ibhagwan** [reply]
Version 0.30 is very old, while it’s possible to make it with with it (a few modifications), it is def better to have the latest fzf.

**Reply by xixiaofinland** [reply]
totally. Just didn't realize my fzf was so old, it worked just fine :D. Thanks for pointing to the right direction :)!
